src/test/scala/nl.knaw.dans.easy.multideposit/SplitMultiDepositAppSpec.scala

```diff
@@ -277,10 +284,32 @@ class SplitMultiDepositAppSpec extends TestSupportFixture with MockFactory with
       }
     }
 
-    "ruimtereis01" should behave like bagContents("ruimtereis01", expectedDataContentRuimtereis01)
-    "ruimtereis02" should behave like bagContents("ruimtereis02", expectedDataContentRuimtereis02)
-    "ruimtereis03" should behave like bagContents("ruimtereis03", expectedDataContentRuimtereis03)
-    "ruimtereis04" should behave like bagContents("ruimtereis04", expectedDataContentRuimtereis04)
+    it should "check report.csv" in {
+      doNotRunOnTravis()
+      
+      reportFile.toJava should exist
+
+      val header :: lines = reportFile.lines.toList
+      val reportLines = lines.collect {
+        case s if s startsWith ruimtereis01 => ruimtereis01 -> s
+        case s if s startsWith ruimtereis02 => ruimtereis02 -> s
+        case s if s startsWith ruimtereis03 => ruimtereis03 -> s
+        case s if s startsWith ruimtereis04 => ruimtereis04 -> s
+      }.toMap
+      // taken from https://stackoverflow.com/a/6640851/2389405
+      val uuidRegex = "[0-9a-f]{8}-[0-9a-f]{4}-[0-9a-f]{4}-[0-9a-f]{4}-[0-9a-f]{12}"
```

Comment:
I see `-[0-9a-f]{4}` repeated 3 times,so: `"[0-9a-f]{8}(-[0-9a-f]{4}){3}-[0-9a-f]{12}"`.

This only matches lower case, would uppercase be valid too?

It would have been fun when we could [predefine] a hex character class.

[predefine]: https://docs.oracle.com/javase/7/docs/api/java/util/regex/Pattern.html

Reply:
> This only matches lower case, would uppercase be valid too?

No, upper-case would not be expected in this case: `UUID.randomUUID()` only provides lower case characters `[a-f]`. As explained on [StackOverflow](https://stackoverflow.com/a/13833488/2389405), there is a [standard](http://www.itu.int/rec/T-REC-X.667/en) that says that a UUID should only use lower-case letters. Then it goes on demonstrating that if you use upper-case in `UUID.fromString`, it will be transformed to lower-case in the end. Hence, I will only match on lower-case here.

Reply:
> I see `-[0-9a-f]{4}` repeated 3 times, so: `"[0-9a-f]{8}(-[0-9a-f]{4}){3}-[0-9a-f]{12}"`.

👍 that is also what [this answer on StackOverflow](https://stackoverflow.com/a/3999043/2389405) suggested in the same question I linked to in the code.
I replaced the original regex with your variant!

Reply:
> It would have been fun when we could predefine a hex character class.

Not sure what you mean by that, but if you found something cool, definitely feel free to contribute 😃